apps/openid-provider-app/package.json

```diff
@@ -19,6 +19,7 @@
   },
   "dependencies": {
     "@azure/cosmos": "^4.0.0",
+    "@azure/functions": "^3.2.0",
```

Comment:
Why are we using version 3?

Reply:
yes, in `openid-provider-app` (not `relying-party-func`) because it is required by `functions-commons`